### ISSUE TITLE
Using medcam with torchio and monai?

### ISSUE THREAD

**Issue by greglahmer** [MECLabTUDA/M3d-Cam#24]
Can this work with the generator and aggregator functions from torchio and the architectures from monai? I have been having problems with establishing gradcam visualizations for my unet with monai alone.

**Comment by Karol-G** [comment]
Yes, it is possible to use with the GridSampler and Aggregator of TorchIO. However, I have no idea if it also works with MONAI as I never worked with that before.
